### PR TITLE
Move js in file

### PR DESCRIPTION
**Pull request by angelozerr**
Move js in file

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
Debugging JS function like fieldChange, initEventListener (to try to understand the communication of message between vscode and html) is not possible because those function are in the sub.html. I think it's th ereason why there are a lot of connsole.log.

This PR moves all JS from stub.html to a stub.js file.

It gives the capability to debug those JS functions with Chrome Debugger.

To do that:

 * open a wizard 
 * open Developer Tools
image
 * Inside Developer Tools, Do Ctrl+O and type stub.js to open it
 * Add a breakpoint 
 * Type something in a field of wizard, you should see the debugger in action

image

File: pages/stub.html [hunk 1]
```diff
@@ -48,182 +46,11 @@ <h2 id="pageTitle" class="section__title section__title--primary"></h2>
     </div>
   </div>
 
-
   <script type="text/javascript">
-    const cssColorRegex = /^(?:(#?)([0-9a-f]{3}|[0-9a-f]{6})|((?:rgb|hsl)a?)\((-?\d+%?)[,\s]+(-?\d+%?)[,\s]+(-?\d+%?)[,\s]*(-?[\d.]+%?)?\))$/i;
```
Comment (robstryker) [inline pages/stub.html:53]: Moving this stuff out is fine to me. +1

**Review by robstryker — APPROVED** [review]
I was trying to avoid the init stuff into the javascript because it was part of the original vscode-page implementation and I kinda wanted to leave that part untouched. 

I suppose the project has changed enough since then (very very drastically) that it's no longer an issue. And since users can't contribute their own stub, I guess it's not important. 

This is a fine patch.

File: pages/stub.html (hunk 1 above)
Comment (angelozerr) [reply pages/stub.html:53]: Glad you like it:)

**Comment by angelozerr** [comment]
Ok I rebased this PR with master, you can merge it.

**Pull request merged**